src/fidesops/service/saas_request/override_implementations/mailchimp_request_overrides.py

```diff
@@ -0,0 +1,136 @@
+
+
+@SaaSRequestOverrideFactory.register(
+    "mailchimp_messages_access", [SaaSRequestType.READ]
+)
+def mailchimp_messages_access(
+    node: TraversalNode,
+    policy: Policy,
+    privacy_request: PrivacyRequest,
+    input_data: Dict[str, List[Any]],
+    identity_data: Dict[str, Any],
+    secrets: Dict[str, Any],
+) -> List[Row]:
+    """
+    Equivalent SaaS config for the code in this function.
+
+    Request params still need to be defined for endpoints with overrides.
+    This is to provide the necessary reference and identity data as part
+    of graph traversal. The resulting values are passed in as parameters
+    so we don't need to define the data retrieval here.
+    path: /3.0/conversations/<conversation_id>/messages
+    request_params:
+    - name: conversation_id
+        type: path
+        references:
+        - dataset: mailchimp_connector_example
+        field: conversations.id
+        direction: from
+    data_path: conversation_messages
+    postprocessors:
+    - strategy: filter
+        configuration:
+        field: from_email
+        value:
+            identity: email
+    """
+    # gather request params
+    conversation_ids = input_data.get("conversation_id")
+
+    # build and execute request for each input data value
+    processed_data = []
+    if conversation_ids:
+        for conversation_id in conversation_ids:
+            try:
+                response = get(
+                    url=f'https://{secrets["domain"]}/3.0/conversations/{conversation_id}/messages',
+                    auth=(secrets["username"], secrets["api_key"]),
+                )
+
+            # here we mimic the sort of error handling done in the core framework
+            # by the AuthenticatedClient. Extenders can chose to handle errors within
+            # their implementation as they wish.
+            except Exception as exc:  # pylint: disable=W0703
+                if config.dev_mode:  # pylint: disable=R1720
+                    raise ConnectionException(
+                        f"Operational Error connecting to Mailchimp API with error: {exc}"
+                    )
+                else:
+                    raise ConnectionException(
+                        "Operational Error connecting to MailchimpAPI."
+                    )
+            if not response.ok:
+                raise ClientUnsuccessfulException(status_code=response.status_code)
+
+            # unwrap and post-process response
+            response_data = pydash.get(response.json(), "conversation_messages")
+            filtered_data = pydash.filter_(
+                response_data, {"from_email": identity_data.get("email")}
+            )
+
+            # build up final result
+            processed_data.extend(filtered_data)
+
+    return processed_data
+
+
+@SaaSRequestOverrideFactory.register(
+    "mailchimp_member_update", [SaaSRequestType.UPDATE]
+)
+def mailchimp_member_update(
+    update_param_values: List[Dict[str, Any]],
```

Comment:
The name `update_param_values` is a bit confusing since we have the concept of `param_values`. At first glance the `for update_params in update_param_values` part makes me think that we're iterating through each param_value. I think we need something that better describes this input as a list of param_values per row to update. Maybe param_values_list, or something along those lines?

Reply:
hmm, i agree the current name is confusing. i'm not sure `param_values_list` clears things up that much though. maybe just being very explicit and `param_values_per_row` ?